static/app/views/organizationGroupDetails/groupActivityItem.tsx

```diff
@@ -323,3 +347,12 @@ function GroupActivityItem({activity, orgSlug, projectId, author}: Props) {
 }
 
 export default GroupActivityItem;
+
+const CodeWrapper = styled('div')`
+  overflow-wrap: anywhere;
+  font-size: ${p => p.theme.fontSizeSmall};
+`;
+
+const StyledCode = styled('span')`
```

Comment:
why not use a `code` tag?

Reply:
I tried a code tag, but since codeowners can potentially be a very long line with no breaks the code block was too aggressive against wrapping

Reply:
I should change the name of this styled component since i'm not using code anymore